app/utils/application-state-utils.ts

```diff
@@ -205,7 +226,11 @@ function getAppealApplicationNextStep(req: Request) {
   }
 
   doThisNextSection.deadline = getDeadline(currentAppealStatus, req);
-
+  if (doThisNextSection.hasOwnProperty('date')) {
```

Comment:
we could use currentAppealStatus is CMA_LISTED instead of adding an extra property.